drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -324,6 +324,13 @@ void at86rf2xx_set_cca_threshold(at86rf2xx_t *dev, int8_t value)
     at86rf2xx_reg_write(dev, AT86RF2XX_REG__CCA_THRES, value);
 }
 
+#ifdef MODULE_OPENTHREAD
+bool at86rf2xx_receiver_listening(at86rf2xx_t *dev)
+{
+    return (at86rf2xx_reg_read(dev, AT86RF2XX_REG__RX_SYN) & 0x80) ? false : true;
+}
+#endif
```

Comment:
this isn't the pkg updated version (I created a separate [branch] for that). That version has a simplified radio layer that doesn't require the LISTENING option

[branch]:https://github.com/jia200x/RIOT/tree/openthread_new